### PR TITLE
refactor

### PR DESCRIPTION
**Pull request by septs**
100% refactor not use docker based solution

1. reduce startup time
2. auto follow latest version

This version, startup time only need 1s (Direct download binary file), current main branch startup time need 3s (Build docker image)

see https://github.com/DimensionDev/Maskbook/runs/5343812830?check_suite_focus=true

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,44 @@
...
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  editorconfig:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: editorconfig-checker/action-editorconfig-checker@main
```
Comment (septs) [inline README.md:38]: suggest, rename this repo name to `setup-checker`

```patch
     runs-on: ubuntu-20.04
     steps:
       - uses: actions/checkout@v2
-      - uses: editorconfig-checker/action-editorconfig-checker@main
+      - uses: editorconfig-checker/setup-checker@main
       - run: editorconfig-checker
```

Can improve the experience

**Comment by septs** [comment]
@mstruebing

File: README.md (hunk 1 above)
Comment (septs) [reply README.md:38]: or new repo?

Comment (mstruebing) [reply README.md:38]: Let's test this at first before renaming/moving the repo. Thank you again very much for your contribution.

**Review by mstruebing — APPROVED** [review]

**Pull request merged**